data/Sri Guru Granth Sahib Ji/0368.json

```diff
@@ -1657,590 +1657,5 @@
         }
       }
     ]
-  },
-  {
-    "id": "P5L",
-    "sttm_id": 1433,
-    "writer": "Guru Ramdas Ji",
-    "section": "Raag Aasaa",
-    "subsection": null,
-    "lines": [
-      {
-        "id": "7PKC",
-        "source_page": 369,
```

Comment:
removed the page number that wasn't suppose to be here. @Sarabveer is this expected?